### PR TITLE
Feature/add cors

### PR DESCRIPTION
**Pull request by GitHubEason**
add cors

### CODE REVIEW

**Review by tatoMa — COMMENTED** [review]
Why do we still need all app.controller / serveice / module and get Hello api?

File: src/app.module.ts [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
+import { CatsModule } from "./cats/cats.module";
+import { MongooseModule } from "@nestjs/mongoose";
 
 @Module({
-  imports: [CatsModule, MongooseModule.forRoot('mongodb://localhost/test')],
+  imports: [
+    ConfigModule.forRoot({ isGlobal: true }),
+    HealthModule,
+    CatsModule,
+    MongooseModule.forRoot("mongodb://localhost/test"),
```
[the remark this answers is not in the document]
Comment (GitHubEason) [reply src/app.module.ts]: Thanks for reminding me, I will put the URL into env, pls check again

File: src/app.module.ts [hunk 2]
```diff
@@ -13,9 +11,7 @@ import { MongooseModule } from "@nestjs/mongoose";
     ConfigModule.forRoot({ isGlobal: true }),
     HealthModule,
     CatsModule,
-    MongooseModule.forRoot("mongodb://localhost/test"),
+    MongooseModule.forRoot(process.env.CATS_URL),
```
Comment (tatoMa) [inline src/app.module.ts:14]: Is it the URL only for the demo cats api? Why it is named CATS_URL?

Comment (GitHubEason) [reply src/app.module.ts]: the URL is changed to DATABASE_URL

**Review by tatoMa — APPROVED** [review]
LGTM

**Review by Jzeva — APPROVED** [review]

**Pull request merged**